bottlerocket/agents/src/bin/eks-resource-agent/eks_provider.rs

```diff
@@ -20,16 +20,21 @@ use resource_agent::provider::{
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
 use std::env::temp_dir;
-use std::fs::write;
 use std::path::Path;
 use std::process::Command;
 use testsys_model::{Configuration, SecretName};
 
+use std::fs::File;
+use std::io::Write;
+
 /// The default region for the cluster.
 const DEFAULT_REGION: &str = "us-west-2";
 /// The default cluster version.
 const DEFAULT_VERSION: &str = "1.24";
```

Comment:
I know this isn't in your diff but we should consider upgrading our default k8s to a newer version. Not needed in this PR but raising here to remind us to get an issue cut to track moving to a newer k8s for default.